pages/common/mpc.md

```diff
@@ -0,0 +1,28 @@
+# mpc
+
+> Music Player Client.
+> Program for controlling Music Player Daemon (MPD).
+
+- Toggle between play and pause:
+
+`mpc toggle`
+
+- Stop playing:
+
+`mpc stop`
+
+- Show information about currently playing song:
+
+`mpc status`
+
+- Play next song:
+
+`mpc next`
+
+- Play previous song:
+
+`mpc prev`
+
+- Seek playing song:
```

Comment:
I thought about "fast-forward" and "rewind", indeed, but I think that could be associated with fast-paced playback in an interactive way, where the user would end the operation once they recognized the part of the song they wanted. But they would work here too, IMO, so I'd say pick whatever you prefer :)